.github/workflows/twister_tests.yml

```
@@ -9,7 +9,7 @@ on:
     - main
     - v*-branch
     paths:
-    - 'scripts/pylib/twister/**'
+    - 'scripts/pylib/**'
```

is it safe to take everything from `scripts/pylib` instead of explicitly choose like this:

```suggestion
    - 'scripts/pylib/twister/**'
    - 'scripts/pylib/build_helpers/**'
```

---

You're right, changed that to explicit `build_helpers` mention.